learning/courses/quantum-diagonalization-algorithms/krylov.ipynb

```diff
@@ -457,7 +457,7 @@
     "$$\n",
     "we see that\n",
     "$$\n",
-    "\\text{span} \\{|R_0\\rangle,|R_1\\rangle,…,|R_k\\rangle \\}=\\text{span} \\{|R_0\\rangle,A|R_0\\rangle,…,A^{k−1}|R_0\\rangle \\}.\n",
+    "\\text{span} \\{|R_0\\rangle,|R_1\\rangle,…,|R_k\\rangle \\}=\\text{span} \\{|R_0\\rangle,A|R_0\\rangle,…,A^{k}|R_0\\rangle \\}.\n",
```

Comment:
I'm not sure why this change is showing up. If you haven't recently, you could try `git pull main` - maybe it isn't realizing properly this change was already made on main.